### PR TITLE
[WIP] Improve onboarding

### PR DESCRIPTION
**Pull request by datn**

### CODE REVIEW

**Comment by jmcarp** [comment]
Need to resolve merge conflicts.

File: AccessList.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Access list for Atlas/AgentQ
```
Comment (mogul) [inline AccessList.md]: I've got mixed feelings on migrating this list out of the onboarding checklist. On one hand, it puts the list in one place so we don't have to repeat risk de-sync'ing the same information in both our onboarding and offboarding processes. On the other, it's harder to generate a comprehensive onboarding/offboarding issue that clearly shows the full status of a person's progress on/off the team when this is in another file.

File: ReadingList.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Reading List
```
Comment (mogul) [inline ReadingList.md]: I've got mixed feelings on migrating these lists out of the onboarding checklist. On one hand, it separates the "stuff to read/watch" from the "stuff to do", which is nice. On the other, it's harder to generate a comprehensive issue that clearly shows the full status of a person's progress onto team when this is in another file.

**Comment by LinuxBozo** [comment]
Closing this in favor of keeping things in the single checklist template.

**Pull request closed without merging**